api/src/context.rs

```diff
@@ -891,9 +891,12 @@ impl Context {
                 address,
                 ledger_info.version(),
                 ledger_info,
-            )?
-            .sequence_number()
-            .saturating_sub(limit as u64)
+            )
+            .map_or(0, |account_resource| {
```

Comment:
So `0` is how we represent the absence of a sequence number?

Reply:
There are two sequence numbers -- An onchain sequence number for the account, and the sequence number used inside a transaction. 

The default onchain sequence number for an account is `0` if the account doesn't have `0x1::Account` resource stored inside it (a stateless account).

For orderless transactions, the default sequence number we use is `u64::MAX`.